specification/glossary.md

```diff
@@ -2,10 +2,130 @@
+
+## CSP
+
+Cloud Service Provider (e.g. Amazon Web Services, Microsoft Azure, Google Cloud Platform).
+
+## Dimension
+
+Specification-defined categorical attribute that provides context or categorization to billing data.
+
+## Effective Cost
+
+A comprehensive and realistic measure of the true cost of a service, inclusive of all reduced rates, discounts, and impacts of amortization.
+
+## Finalized Tag
+
+A tag with one tag value chosen from a set of possible tag values after being processed by a set of provider-defined or user-defined rules.
+
+## FOCUS
+
+FinOps Cost and Usage Specification - An effort to build an open specification for cloud billing data.
+
+## Interruptible
+
+Category of compute resources that can be paused or terminated by the CSP within certain criteria, often available for discounted pricing when compared to the equivalent non-interruptible resource.
+
+## List Unit Price
+
+Provider published unit price for a single pricing measurement unit.
+
+## Metric
+
+A quantifiable, standardized measure used to assess, evaluate, or track the performance, quality, or characteristics of a system, process, or service.
+
+## MSP
+
+Managed Service Provider - A company or organization that provides outsourced management and support of a range of IT services, such as network infrastructure, cybersecurity, cloud computing, and more.
+
+## On-Demand
+
+A term that describes a service that is available and provided immediately or as needed, without requiring a pre-scheduled appointment or prior arrangement. In Cloud Computing, virtual machines can be created and terminated as needed, i.e. on demand.
+
+## Practitioner
+
+An individual whose charter is to enhance business value from cloud infrastructure, by balancing innovation, velocity, and cost efficiency.
+
+## Provider
+
+An entity that delivers services to consumers or clients i.e. a company that offers cloud computing, managed IT services, web hosting, or other technology-related services.
+
+## Price List
+
+Comprehensive list of prices for offerings from a CSP, published by the CSP.
+
+## Resource
+
+An provisionable infrastructure component that is available to a customer in a cloud environement e.g. virtual machine, storage volume etc.
+
+## Row
+
+An entry in a billing data set representing the consumption of a resource or other measurable activity.
+
+## Service
+
+A capability defined and offered by cloud service providers to satisfy certain needs, often through activation of resources e.g. the capability of creating a virtual machine or a storage volume are services.
+
+## SKU
+
+Construct composed of the common properties of a product offering associated with one or many SKU Prices.
+
+## SKU Price
+
+Unit price used to calculate a charge. Any SKU Price can only be associated to one SKU.
```

Comment:
Should we mention that "this can be referenced on a price list published by a provider to look up detailed information, including a corresponding list unit price. The composition of the properties associated with the SKU Price may differ across providers. "

Reply:
Added aspects of this comment